.github/workflows/pr-reviews.yml

```
@@ -1,21 +1,51 @@
 name: "PR Reviews CI"
 
-on: [pull_request]
+on: 
+  # Chaos ahead, PROCEED AT YOUR OWN RISK!
+  pull_request_target:
+    types: [
+      assigned,
+      unassigned,
+      labeled,
+      unlabeled,
+      edited,
+      opened,
+      synchronize,
+      locked,
+      unlocked,
+      review_requested,
+      review_request_removed
+    ]
```

The rendering looks kinda sus from GitHub, how about this?

```suggestion
on: 
  # Chaos ahead, PROCEED AT YOUR OWN RISK!
  pull_request_target:
    types: 
      - assigned
      - unassigned
      - labeled
      - unlabeled
      - edited
      - opened
      - synchronize
      - locked
      - unlocked
      - review_requested
      - review_request_removed
```